### PR TITLE
flatpak: Copy vls and deps from the vala sdk

### PR DESCRIPTION
**Pull request by lw64**

### CODE REVIEW

**Comment by lw64** [comment]
Not sure if its related, but vls cannot find gtk4 and workbench namespaces anymore.

File: re.sonny.Workbench.Devel.json [hunk 1]
```diff
@@ -148,6 +99,11 @@
           "type": "dir",
           "path": "."
         }
+      ],
+      "post-install": [
+        "cp -a /usr/lib/sdk/vala/bin/* /app/bin/",
```
Comment (sonnyp) [inline re.sonny.Workbench.Devel.json:104]: > cp: cannot stat '/usr/lib/sdk/vala/bin/*': No such file or directory

can you use `install` ?

Comment (lw64) [reply re.sonny.Workbench.Devel.json:104]: This is strange. it seems like the vala sdk is not mounted into the extension point??

Comment (sonnyp) [reply re.sonny.Workbench.Devel.json]: This is a flatpak-vscode only - I fixed it in this commit https://github.com/sonnyp/Workbench/pull/273/commits/e05ff73b08cdd15e38b05d5e66f14a48c7d6016f

**Comment by sonnyp** [comment]
> Not sure if its related, but vls cannot find gtk4 and workbench namespaces anymore.

I can reproduce, definitely related. Could you look into it?

**Review by sonnyp — APPROVED** [review]
I also cannot reproduce the issue anymore.

**Pull request merged**